admin/developers_tool_kit.php

```diff
@@ -20,6 +20,10 @@
   $configuration_key_lookup = (isset($_POST['configuration_key'])) ? $_POST['configuration_key'] : '';
   $q_const = $q_func = $q_class = $q_tpl = $q_all = '';
 
+  // if the $_POST['wrap_lines'] is set, then we'll use the .wrappable CSS class for the output.
+  $wrap_lines = isset($_POST['wrap_lines']) && (int)$_POST['wrap_lines'] > 0 ?: false;

```

Comment:
Argh, somehow the tick box for `wrap_lines` input field didn't get into this commit.